jobs/build/sweep/Jenkinsfile

```diff
@@ -59,7 +59,10 @@ node {
     stage("Repair bug state") {
         try {
             currentBuild.description += "* Moving attached bugs in MODIFIED state to ON_QA...\n"
-            buildlib.elliott "--group=openshift-${version} repair-bugs --use-default-advisory ${kind} --auto" 
+            retry (3) {
+                buildlib.elliott "--group=openshift-${version} repair-bugs --use-default-advisory ${kind} --auto"
+                sleep(time: 1, unit: 'MINUTES')
```

Comment:
Would be nice not to wait when it succeeds.

Reply:
Whoops! That's a good point.